zulipterminal/core.py

```diff
@@ -246,11 +255,11 @@ def exit_popup(self) -> None:
         self.loop.widget = self.view
 
     def show_help(self) -> None:
-        help_view = HelpView(self, "Help Menu (up/down scrolls)")
+        help_view = HelpView(self, f"Help Menu {SCROLL_PROMPT}")
         self.show_pop_up(help_view, "area:help")
 
     def show_markdown_help(self) -> None:
-        markdown_view = MarkdownHelpView(self, "Markdown Help Menu (up/down scrolls)")
+        markdown_view = MarkdownHelpView(self, "Markdown Help Menu " + SCROLL_PROMPT)
```

Comment:
> I've not tested this yet, but you updated the formatting of other lines but not this one?

It's the same formatting, just didn't use f-strings for this one deliberately because if I did, it exceeds the recommended line length. But, consistency is more important than minimizing LOC, I'll turn that back into an f-string.